### PR TITLE
Slider

### PR DESCRIPTION
**Pull request by davrivas**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -16,31 +15,73 @@
...
-                <li class="sliderButton deactive"></li>
+                <li class="sliderButton sliderActive"></li>
+                <li class="sliderButton"></li>
+                <li class="sliderButton"></li>
             </ul>
-
-            <div class="sliderGradient"></div>
         </div>
 
         <script type="text/javascript" src="static/js/slider.js"></script>
```
Comment (cavilarts) [inline index.html]: You should move this at the bottom of the body

File: static/js/slider.js [hunk 2]
```diff
@@ -7,13 +7,13 @@
...
         for (i = 0; i < $sliderList.length; i++) {
             $sliderList[i].style.width = windowWidth + "px"; 
-        }
-
-        $sliderImages.style.width = windowWidth * howManyChildren + "px";
-    }
+         }
+ 
+         $sliderImages.style.width = windowWidth * howManyChildren + "px";
+     }
```
Comment (cavilarts) [inline static/js/slider.js]: Wich is better, the simple assignation or the loop

File: static/js/slider.js [hunk 3]
```diff
@@ -26,15 +26,47 @@
...
+            button;
+
+        for (i = 0; i < $sliderList.length; i++) {
+            $sliderList[i].style.width = windowWidth.toString() + "px";
+            console.log($sliderList[i].style.width);
+        }
+
+        $sliderImages.style.width = (windowWidth * $sliderList.length) + "px";
+        
+        /* lo que uiqero hacer es recorrer todos los elementos de los botones del slider, y de acuerdo al indice en el que se encuentre el boton con clase "sliderActive", se se aplica el estilo del slider images de acuerdo al siguiente indice (esto solo es una gúia): <$sliderImages.style.left = (windowWidth * (index * (-1))) + "px";>
```
Comment (cavilarts) [inline static/js/slider.js]: Why this comment is here?

File: static/js/slider.js [hunk 4]
```diff
@@ -26,15 +26,47 @@
...
+
+        Y por esa razón propongo este ciclo::::::
+        for (i = 0; i < sliderButtons.length; i++) {
+            if (sliderButtons[i].className === "sliderActive") {
+                console.log(i);
+            }
+        }
+
+        aun asi esto no estodo :/ */
+
```
Comment (cavilarts) [inline static/js/slider.js]: What you should do to achieve this?

File: static/js/slider.js (hunk 3 above)
Comment (davrivas) [inline static/js/slider.js]: sorry it's a mental note i forgot delete it.

File: static/js/slider.js (hunk 4 above)
Comment (davrivas) [inline static/js/slider.js]: Well it's an idea, but i forgot to delete that

**Pull request closed without merging**